larrecodnn/ImagePatternAlgs/NuSonic/Triton/TritonData.h

```diff
@@ -0,0 +1,125 @@
+#ifndef NuSonic_Triton_TritonData
+#define NuSonic_Triton_TritonData
+
+//#include "FWCore/Utilities/interface/Exception.h"
+//#include "FWCore/Utilities/interface/Span.h"
+#include "canvas/Utilities/Exception.h"
+#include "larrecodnn/ImagePatternAlgs/NuSonic/Triton/Span.h"
+
+#include <vector>
+#include <string>
+#include <unordered_map>
+#include <numeric>
+#include <algorithm>
+#include <memory>
+#include <any>
+
+#include "grpc_client.h"
+#include "grpc_service.pb.h"
+
+//forward declaration
+class TritonClient;
+
+//aliases for local input and output types
+template <typename DT>
+using TritonInput = std::vector<std::vector<DT>>;
+template <typename DT>
+using TritonOutput = std::vector<triton_span::Span<const DT*>>;
+
+//store all the info needed for triton input and output
+template <typename IO>
+class TritonData {
```

Comment:
Recommend placing all of this in a namespace (e.g. `lartriton`).

Reply:
done